tests/device_tests/test_autolock.py

```diff
@@ -92,6 +93,41 @@ def test_apply_minimal_auto_lock_delay(client):
         get_test_address(client)
 
 
+@pytest.mark.parametrize(
+    "seconds,expected_error",
```

Comment:
please don't write tests parametrized on whether they succeed of fail. this should be two test cases.

please split into two, one for failing cases and one for succeeding cases.

also you can remove all `expected_responses` from here, there is nothing interesting to check for them

Reply:
mark the failing test with `pytest.mark.skip_ui`

in the succeeding test, please add a case where the UI says "minutes", so that we have a UI recording of that

Reply:
Done 8516e7a.